src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -123,12 +123,13 @@ public void bindArmControls() {
 				new ManualArmOverrideOnCommand(
 						s.armSubsystem, codriveController::getRightY, codriveController::getLeftY));
 		armManualControlOff.onTrue(new ManualArmOverrideOffCommand(s.armSubsystem));
-		armLowButton.onTrue(new SetFullArmCommand(s.armSubsystem, ARM_LOW_POSITION, WRIST_RETRACT));
+		armLowButton.onTrue(
+				new SetFullArmCommand(s.armSubsystem, ARM_LOW_POSITION, WRIST_RETRACT, 0.5));
```

Comment:
The `0.5` here and below ought to be a named constant so it's clear to code readers what it's for